src/main/java/com/example/TwoRoomsBoom/models/role/BlueTeam.java

```diff
@@ -0,0 +1,14 @@
+package com.example.TwoRoomsBoom.models.role;
+
+import com.example.TwoRoomsBoom.models.Role;
+
+public class BlueTeam extends Role{
```

Comment:
Just so I know, are the normal team players just called Blue and Red team? Are they not called like agents or anything? Just curious

Reply:
Yes, the normal team players are just called Blue and Red team. Agents are completely different roles.